### ISSUE TITLE
Maps do not work

### ISSUE THREAD

**Issue by ShimShamSam** [fourseven/meteor-scss#44]
Create a file with the following:

````
$my_map: (
    a: 1,
    b: 2
);
```

The error reported is:
```
While building the application:
test.scss: Scss compiler error: undefined
/path/to/test.scss:2: error: error reading values after a
```

**Comment by keyvanfatehi** [comment]
@fourseven are source maps working yet?

**Comment by fourseven** [comment]
hi @keyvanfatehi - no they're not, libsass still has issues generating them, which look to be improved in v3.2. node-sass is using v3.1 of libsass in their 2.0.0 beta, so it might be a while yet unfortunately.

**Comment by rizkysyazuli** [comment]
too bad. i wanted to ask this as well.

although it seems to be fixed in libsass 3.1.0-beta
http://sassmeister.com/gist/511dee5bbd1a40054d92

btw, a bit OOT, but i tried switching to [grigio:ruby-sass](https://atmospherejs.com/grigio/ruby-sass). but i'm getting this error even though i already removed your package.

    While building the application:
    error: conflict: two packages included in the app, grigio:ruby-sass and
    fourseven:scss, are both trying to handle .scss
    error: conflict: two packages included in the app, grigio:ruby-sass and
    fourseven:scss, are both trying to handle .sass
